cypress/integration/home_spec.js

```diff
@@ -0,0 +1,21 @@
+describe('Home page Search tests', () => {
+  it('should search shabad Jo Mange Thakur Apne Te with default settings', () => {
+    cy.visit('/')
+
+    cy.get('#search')
+      .clear()
+      .type('bhb')
+
+    // get api response
+    cy.wait(3000)
```

Comment:
@plug-n-play This is considered kinda antipattern in cypress maybe we can do something else here?